pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -55,6 +55,13 @@ const (
 	// VirtualListenerName is the name for traffic capture listener
 	VirtualListenerName = "virtual"
 
+	// VirtualListenerName is the name for traffic capture listener
+	VirtualInboundListenerName = "virtualInbound"
+
+	// A dedicated inbound traffic capture port for sidecar proxy.
+	// See also `MeshConfig.ProxyListenPort`.
+	ProxyInboundListenPort = 15006
```

Comment:
This is not good. And will likely break if this is used by something else. This should be a config option and not a hardcoded setting.

Reply:
I don't think we ever tested changes of 15001 port - or ever changed it. 

We already have too many untested and confusing settings in values.yaml - we can allow some env variable to override this if an advanced user really wants, but we can't expose every small implementation detail as a user setting.

Reply:
hard code 15006 in code